packages/renderer/src/lib/preferences/PreferencesConnectionCreationRendering.svelte

```diff
@@ -194,7 +194,11 @@ async function handleValidComponent() {
     data[key] = value;
   }
 
-  connectionAuditResult = await window.auditConnectionParameters(providerInfo.internalId, data as AuditRequestItems);
+  try {
+    connectionAuditResult = await window.auditConnectionParameters(providerInfo.internalId, data as AuditRequestItems);
+  } catch (e: any) {
```

Comment:
I don't think we should use any here as message might not be defined, it's unknown type by default

in some code we had to fix it as we had undefined or null then e.message was throwing an error

```typescript
} catch (err: unknown) {
  if (err instanceof Error) {
  ...
  }
}
```